ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -18,4 +19,11 @@ import_config:
     # A map from versions to dump and their output names.
     versions_to_import:
       123: testnet_v1.json
+# Config to generate the transactions via localnode.
+script_transaction_generator_config:
+  scripted_transactions:
```

Comment:
How does the config handle generation of different transactions? Could add an example here to to demonstrate

Reply:
Also I remember we talked about renaming "steps" to "transactions" to be clarify what the output of this is. It's still not clear to me if a transaction has multiple steps, or if there's 1 transaction generated per step?

Reply:
done; updated.